### ISSUE TITLE
Should we send email notifications about deleted topics?

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#2571]
Now `tc-notifications` supports notifications about deleted topics `notifications.connect.project.topic.deleted`. And when some topic is being deleted we get notifications by email that topic was deleted:

![image](https://user-images.githubusercontent.com/146016/46296710-95faa880-c5cd-11e8-84d0-33adbc5dc74b.png)

There is a `View post on Connect` button which is a link to deleted topic `https://connect.topcoder.com/projects/5562#feed-27377`. When we click it the dashboard page will be opened but there wouldn't be deleted topic to show.

Is it desired behaviour?

fyi @RishiRajSahu

**Comment by vikasrohit** [comment]
@maxceem I think it is okay to redirect to the topic at least because user would know that a user has deleted a message from that topic. However, I would like to defer it to the product team @vic-topcoder @acshields .

**Comment by maxceem** [comment]
> I think it is okay to redirect to the topic

@vikasrohit yes, but I think this notification is about deleted **topic** not **post**. Not sure if we can show user something in this case.
